pkg/trace/config/config.go

```diff
@@ -135,6 +137,9 @@ func New() *AgentConfig {
 		ReceiverPort:    8126,
 		MaxRequestBytes: 50 * 1024 * 1024, // 50MB
 
+		ReceiverReadTimeout:  time.Duration(5) * time.Second,
+		ReceiverWriteTimeout: time.Duration(5) * time.Second,
```

Comment:
I think the case here is not needed
```suggestion
		ReceiverReadTimeout:  5 * time.Second,
		ReceiverWriteTimeout: 5 * time.Second,
```